pkg/artifactory/datasource/security/datasource_artifactory_permission_target_test.go

```diff
@@ -0,0 +1,143 @@
+package security_test
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/jfrog/terraform-provider-artifactory/v6/pkg/acctest"
+	"github.com/jfrog/terraform-provider-artifactory/v6/pkg/artifactory/resource/repository"
+	"github.com/jfrog/terraform-provider-artifactory/v6/pkg/artifactory/resource/security"
+	"github.com/jfrog/terraform-provider-artifactory/v6/pkg/artifactory/resource/user"
+	"github.com/jfrog/terraform-provider-shared/test"
+	"github.com/jfrog/terraform-provider-shared/util"
+	"log"
+	"testing"
+)
+
+func deletePermissionTarget(t *testing.T, name string) error {
+	restyClient := acctest.GetTestResty(t)
+	_, err := restyClient.R().Delete(security.PermissionsEndPoint + name)
+
+	return err
+}
+
+// TODO: Where can we put this user code
+func deleteUser(t *testing.T, name string) error {
```

Comment:
You can put these shared test utility funcs here: https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/acctest/test.go